app/controllers/api/v2/trade_leads/consolidated_controller.rb

```diff
@@ -1,4 +1,4 @@
 class Api::V2::TradeLeads::ConsolidatedController < Api::V2Controller
   include Searchable
-  search_by :countries, :industries, :q, :sources, :publish_date, :end_date, :publish_date_amended
+  search_by :countries, :industries, :q, :sources, :publish_date, :end_date, :publish_date_amended, :categories
```

Comment:
I'm not sure if we'll actually need to support a query param for searching categories. I'd like to see the Google doc first though (access pending).